# Функция plus_two() выполняет одну простую задачу — выводит результат сложения
# переданного в нее числа 2 и значения переменной number. В переменную number должно быть
# передано число. Обработайте ситуацию, если в эту переменную переданы данные какого-то
# другого типа. В случае ошибки напечатайте в консоли сообщение «Ожидаемый тип данных
# — число!».
# Запустите код и проверьте работу кода в консоли.
# Подсказка:
# Используйте конструкцию try/except.В процессе поиска решения попробуйте вывести в
# консоль сумму строки и числа, изучите сообщение об ошибке.В Python есть специальное
# исключение для ситуации, если тип переданного значения не соответствует ожиданиям

def plus_two():
    try:
        number = input("Введите любое число: ")
        result = 2 + int(number)
        print(result)
    except ValueError:
        print("Ожидаемый тип данных — число!")
        result = str(number) + str(2)
        print(result)

plus_two()
